pyanaconda/product.py

```diff
@@ -50,6 +50,10 @@
     productArch = productStamp[productStamp.index(".") + 1:]      # pylint: disable=no-member
 if productVersion == "development":
     productVersion = "rawhide"
+# 8.0.X -> 8.0
+if productVersion.count('.') == 2:
```

Comment:
Maybe it could be `>= 2`. Just in case. The remaining code should still work fine.

Reply:
Good point.

Reply:
Agreed, updated